docs/parameters/help_text.md

```diff
@@ -63,17 +63,7 @@ nextflow run my_pipeline --help outdir
 ## Hidden parameters
 
 Params that are set as `hidden` in the JSON Schema are not shown in the help message.
-To show these parameters, set the `validationShowHiddenParams` parameter:
-
-```bash
-nextflow run my_pipeline --help --validationShowHiddenParams
-```
-
-or specifying it in the configuration file
-
-```groovy
-params.validationShowHiddenParams = true
-```
+To show these parameters, set the `validation.showHiddenParams = true` configuration option
```

Comment:
Out of interest would `--validation.showHiddenParams` work from the command line?

Reply:
Nope that doesn't work, and as far as I know you can't change config values on the command line (I've tried...)

Reply:
This was one that I was a bit conflicted about making a config value, but I really wanted to reduce the amount of custom parameters and `--help` seemed to be the only one that absolutely needed to be a parameter.

Reply:
That is sad. But makes sense as I guess nf would struggle to know if a param was for `params` or a config value

Reply:
Indeed! Maybe later we can still make a parameter for this value, but I'd rather see first if that's something the community actually wants. Let's get this beast out first :grin: